api/profile/profileModel.js

```diff
@@ -51,6 +51,23 @@ const findOrCreateProfile = async (profileObj) => {
   }
 };
 
+async function mentorApplicationData(profile_id) {
```

Comment:
These model functions look good. Just curious, cause there are functions in the applications router that return a similar set of data, is there a reason you decided to use these instead?